docs(attraqt):Added documentation for Attraqt's recommendation loader (#541)
front-commerce/developers.front-commerce.com: docs/magento2/search-engine.mdx

If you are using the default theme or the Chocolatine theme, the search bar
should now be visible.

## Attraqt recommendations

> _Since version `2.19`_

Front-Commerce provides a loaders to retrieve recommendations from
[Attraqt's Widgets](https://attraqt.gitbook.io/developer-documentation/xo-recommendations/readme-first).

### Front-Commerce configuration

> On Front-Commerce side, you need to enable the Attraqt recommendation module by
> making the following changes in your .front-commerce.js file:

Reviewer: **nitpicking**

```suggestion
To enable the Attraqt recommendation module, you need to add it to your `.front-commerce.js` config:
```

```js title=".front-commerce.js"
module.exports = {
  // highlight-next-line
  modules: ["./node_modules/front-commerce/modules/recommendations-attraqt"],
  serverModules: [

>     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },

Reviewer: Does this actually exist? otherwise I will not add it here as to avoid confusion.

Author: I don't think it exists actually, I took the same base as the snippets above in this file. I'll fix it !

_Note: It seems the "FrontCommerceCore" module has been already used a good number of times throughout the doc already 😕_

Reviewer: I think it should be removed or fixed (in a seperate mr)

    // highlight-start
    {
      name: "AttraqtRecommendations",
      path: "recommendations-attraqt/server/modules/attraqt-recommendations",
    },
    // highlight-end
    { name: "Magento2", path: "server/modules/magento2" },
  ],
};
```

> Then, in your `.env` file, you need to define the
> `FRONT_COMMERCE_ATTRAQT_RECOMMENDATION_API_URL` environment variable:

Reviewer: **Nitpicking:** The file name (`.env`), and the environment variable name and value is also included in the code block, so repeating it here makes the docs harder to read.

```suggestion
Then, you need to define the API url in your environment variables.
```
